R/cnd-utils.R

```diff
@@ -0,0 +1,71 @@
+#' Signal a deprecated or obsolete function
+#'
+#' @description
+#'
+#' These functions signal a function as deprecated or obsolete.
```

Comment:
I'd rather use the same verb here, even though the actions are slightly different.

I think we should also use a suffix `signal_dep_fun()` so that we can naturally extend to `signal_dep_arg()`.

I'd also like to consider capturing the date or package version somehow, so that we can have a standard soft dep -> dep -> defunct -> deleted process.